go.mod

```diff
@@ -14,13 +14,14 @@ require (
 	github.com/opencontainers/image-spec v1.1.0-rc6
 	github.com/pbnjay/memory v0.0.0-20210728143218-7b4eea64cf58
 	github.com/pterm/pterm v0.12.79
+	github.com/stretchr/testify v1.8.4
```

Comment:
Can we communicate this to the rest of the team.

Reply:
@colesnodgrass @perangel I used testify/assert to assert a list of values.

Reply:
I've been avoiding using the testify package as I don't think it's necessary for most situations, and it feels a little overkill to bring it in for one function.

Most of the code in `abctl` uses the `cmp` package and we can easily use that package for verifying the two lists match
```go
func compareList(t *testing.T, expect, actual []string) {
	t.Helper()
	sort.Strings(expect)
	sort.Strings(actual)
	if d := cmp.Diff(expect, actual); d != "" {
		t.Error(d)
	}
}
```

Reply:
Is it really causing any trouble? It's handy and popular.

Reply:
While I'm personally not a huge fan of testify (I also haven't looked at in in years, so maybe it is worth taking another look with older eyes).  If everyone else wants to start using it, I'll be fine with that. I care mostly about simplicity and consistency.